### PR TITLE
Update to initial preview7 manifest.

### PR DESCRIPTION
**Pull request by crummel**
core-sdk build preview7-012802, SHA ad21996be34bafd85f484acbda73643057db4b01.

### CODE REVIEW

**Comment by crummel** [comment]
With this last fix I'm expecting smoke-test to pass and correct prebuilts on the production build.  I believe I have a fix for the Windows issue and I'll be updating the offline baseline once that build is done.  I think it's possible for reviewing to start now.

File: patches/cli/0002-Consolidate-Newtonsoft.Json-versions.patch [hunk 1]
```diff
@@ -1,17 +1,17 @@
-From 860656146b1d7d882daf3ab6be3607604ab2d4e7 Mon Sep 17 00:00:00 2001
+From bd13401dcc8b22047ea57253620dc978c9a2bc44 Mon Sep 17 00:00:00 2001
 From: Chris Rummel <crummel@microsoft.com>
 Date: Mon, 11 Feb 2019 12:56:51 -0600
-Subject: [PATCH 4/7] Consolidate Newtonsoft.Json versions.
+Subject: [PATCH 2/6] Consolidate Newtonsoft.Json versions.
```
Comment (dseefeld) [inline patches/cli/0002-Consolidate-Newtonsoft.Json-versions.patch]: What is this patch doing?  It doesn't look like it's changing anything.

Comment (crummel) [reply patches/cli/0002-Consolidate-Newtonsoft.Json-versions.patch]: Removed this patch.

**Comment by crummel** [comment]
Merging with Windows and OSX failing.  Logged https://github.com/dotnet/source-build/issues/1153 for Windows, OSX is a Mac-specific prebuilt issue that should be fixed in my latest commit.

**Pull request merged**